Source/WebKit/GPUProcess/media/RemoteLegacyCDMFactoryProxy.cpp

```diff
@@ -190,9 +190,10 @@ bool RemoteLegacyCDMFactoryProxy::allowsExitUnderMemoryPressure() const
 const Logger& RemoteLegacyCDMFactoryProxy::logger() const
 {
     if (!m_logger) {
-        m_logger = Logger::create(this);
+        Ref logger = Logger::create(this);
+        m_logger = logger.ptr();
         auto connection = m_gpuConnectionToWebProcess.get();
-        m_logger->setEnabled(this, connection ? connection->sessionID().isAlwaysOnLoggingAllowed() : false);
+        logger->setEnabled(this, connection ? connection->sessionID().isAlwaysOnLoggingAllowed() : false);
```

Comment:
`connection ? connection->sessionID().isAlwaysOnLoggingAllowed() : false` could be `connection && connection->sessionID().isAlwaysOnLoggingAllowed()`

Reply:
Fixed.